### PR TITLE
Navbar integration, devise views to haml, edit profile link, last login and update times added

### PR DESCRIPTION
**Pull request by coolRaider**
Navbar added, devise views changed from erb to haml, edit profile link added to navbar, last login, updated time for article and created time for comments added

### CODE REVIEW

File: app/assets/stylesheets/articles.scss [hunk 1]
```diff
@@ -14,45 +23,40 @@
   @extend .alert, .alert-success;
 }
 
-.container-fluid {
+body{
```
Comment (joney000) [inline app/assets/stylesheets/articles.scss:26]: can this styling be replace by bootstrap designing?

File: app/views/articles/_form.html.haml [hunk 2]
```diff
@@ -1,24 +1,25 @@
...
+    .form-group
+      %p
+        = f.label :text
+        %br
+        = f.text_area :text, :class => "form-control input-sm", :rows => "8"
 
-    %p
-      = f.submit nil, :class => "mysubmit"
+      %p
+        = f.submit nil, :class => "my-submit"
```
Comment (joney000) [inline app/views/articles/_form.html.haml:25]: add a line at end.

File: app/views/articles/index.html.haml [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 = link_to 'New article', new_article_path
 
-%table.mytable
+%table.my-table
```
Comment (joney000) [inline app/views/articles/index.html.haml:5]: can you replace this with bootstrap table style.

File: app/views/devise/mailer/unlock_instructions.html.haml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+%p
+  Hello #{@resource.email}!
+%p Your account has been locked due to an excessive number of unsuccessful sign in attempts.
+%p Click the link below to unlock your account:
+%p= link_to 'Unlock my account', unlock_url(@resource, unlock_token: @token)
```
Comment (joney000) [inline app/views/devise/mailer/unlock_instructions.html.haml:5]: again empty lines.

File: app/views/layouts/application.html.haml [hunk 5]
```diff
@@ -5,11 +5,25 @@
...
+          - if user_signed_in?
+            %p.navbar-text.small=  "Signed in as ".concat current_user.email.concat ", last login at ".concat current_user.last_sign_in_at.to_s
+            = link_to 'Logout', destroy_user_session_path, method: :delete, :class => "btn btn-default navbar-btn"
+  
+  .my-container-fluid
+    - flash.each do |type, msg|
+      .alert{:class => "alert-#{type}"}
+        = msg
+        %button.close{ "data-dismiss" => "alert"} x
+    = yield
```
Comment (joney000) [inline app/views/layouts/application.html.haml:29]: again should be empty line.

File: app/views/comments/_comment.html.haml [hunk 6]
```diff
@@ -1,5 +1,5 @@
 %p
   %blockquote.comment
     %em= comment.body
-    %footer= comment.commenter
+    %footer= comment.commenter.concat " commented at ".concat comment.updated_at.to_s
```
Comment (alienware) [inline app/views/comments/_comment.html.haml]: I recommend using string interpolations.

`"#{comment.commenter} commented at #{comment.updated_at.to_s}"`

**Comment by alienware** [comment]
Looks good by code, without running to check the changes.

File: app/assets/stylesheets/articles.scss (hunk 1 above)
Comment (coolRaider) [inline app/assets/stylesheets/articles.scss:26]: I have added body top margin here as on bootstrap page. I have tried to abstract the classes so I could apply the same style throughout the website for identical tags. I'll see if I could reduce the customisations to a minimum.

File: app/views/articles/index.html.haml (hunk 3 above)
Comment (coolRaider) [inline app/views/articles/index.html.haml:5]: It is applying multiple bootstrap classes in the background. Wanted to keep it readable and consistent across the website.

**Comment by coolRaider** [comment]
done the required changes, merging with master

**Pull request merged**